numpy/lib/tests/test_utils.py

```diff
@@ -56,11 +56,24 @@ def test_safe_eval_nameconstant():
     utils.safe_eval('None')
 
 
-def test_byte_bounds():
-    a = arange(12).reshape(3, 4)
-    low, high = utils.byte_bounds(a)
-    assert_equal(high - low, a.size * a.itemsize)
-
+class TestByteBounds(object):
+
+    def test_byte_bounds(self):
+        a = arange(12).reshape(3, 4)
+        low, high = utils.byte_bounds(a)
+        assert_equal(high - low, a.size * a.itemsize)
```

Comment:
I think this could do with a comment explaining that this is because the array is contiguous

Reply:
added